kratos/solving_strategies/builder_and_solvers/builder_and_solver.h

```diff
@@ -236,7 +238,7 @@ class BuilderAndSolver
 
         mReshapeMatrixFlag = false; //by default the matrix is shaped just once
         //		mVectorsAreInitialized = false;
-
+        mCalculateReactionsFlag = false;
```

Comment:
I think it would be better to initialize the member to False rather than doing it separately in every constructor
=> If someone adds a new constructor the same problem might appear again

Reply:
Should I do this for the other variables as well? I just followed the style of the file.

Reply:
> Should I do this for the other variables as well? I just followed the style of the file.

+1 from my side
but maybe wait for other comments

Reply:
What's the policy for this @pooyan-dadvand , @RiccardoRossi ?

Reply:
ping @pooyan-dadvand @RiccardoRossi . What to do in these cases?

Reply:
I would suggest going ahead and initializing them whenever possible. I guess that the only motive it is not done yet is that this was new in C++11.